.github/workflows/build-release.yaml

```
@@ -11,9 +11,20 @@ jobs:
     steps:
       - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)